Content.Server/Cuffs/CuffableSystem.cs

```diff
@@ -39,5 +41,13 @@ private void OnCuffableGetState(EntityUid uid, CuffableComponent component, ref
             // the iconstate is formatted as blah-2, blah-4, blah-6, etc.
             // the number corresponds to how many hands are cuffed.
         }
+
+        private void OnBuckleAttemptEvent(EntityUid uid, CuffableComponent component, ref BuckleAttemptEvent args)
+        {
+            if (component.CuffedHandCount > 0)
```

Comment:
you should still be able to unbuckle if you have a free hand, just compare if CuffedHandCount equals the hand count

Reply:
Fixed